subsys/bluetooth/host/Kconfig

```diff
@@ -590,3 +590,23 @@ config BT_PAGE_TIMEOUT
 	  (N * 0.625) ms.
 
 endif # BT_BREDR
+
+config BT_HCI_VENDOR_EVENT_SUPPORT
+	# Virtual/hidden option to make the conditions more intuitive
+	bool
+
+config BT_HCI_VENDOR_EVENT_STACK
```

Comment:
I don't think we need this one at all, we already have `BT_HCI_VS_EXT`, and so that could be enabled by default since soon we will handle VS events.

Reply:
Hmm not sure about the BT_HCI_VS_EXT.
According to the help text that is for some special HCI commands.

Also looking at BT_HCI_MESH_EXT which i guess would be a user of VS events, it doesn't even depend on BT_HCI_VS_EXT, how does that fit together?

Reply:
We can update the text. For me, `BT_HCI_VS_EXT` enables handling of the `0xff` event. After that, we need a new configuration option for the handling of additional VS events that are not part of the Zephyr set. I would propose that this is: `BT_HCI_VS_EVT_HOOK`

Reply:
A bit more clarification:

* Zephyr VS commands and events:  the ones defined in https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/reference/bluetooth/hci.txt
* User VS commands: Additional commands and events defined by the user

`BT_HAS_HCI_VS`: Controller supports Zephyr VS commands and events
`BT_HCI_VS`: Host and Controller enable support for Zephyr VS commands and events
`BT_HCI_VS_EXT`: Host and Controller enable support for extended Zephyr VS commands and events
`BT_HCI_VS_EXT_DETECT`: Auto-detect if the controller supports Zephyr VS commands and events

`BT_HCI_VS_EVT_USER`: Add support for registering a user event callback